client-tez/src/main/java/org/apache/tez/runtime/library/common/sort/impl/RssTezSpillRecord.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.tez.runtime.library.common.sort.impl;
```

Comment:
Must we sort in the map side?

Reply:
tez output is sort，i think yes

Reply:
We don't spill in the map side in the mr's implement.

Reply:
I refer to the mr's method  'org.apache.hadoop.mapred.SortWriteBuffer#getData' . 
And there is no spill here, this class RssTezSpillRecord is used for statistics each partition's records and generate events will use it.
Is it also need name change?

Reply:
> I refer to the mr's method 'org.apache.hadoop.mapred.SortWriteBuffer#getData' . And there is no spill here, this class RssTezSpillRecord is used for statistics each partition's records and generate events will use it. Is it also need name change?

OK, If we don't spill, I think we should change the name and remove the word `Spill`.

Reply:
ok, i will change it.